usaspending_api/awards/models.py

```diff
@@ -455,7 +455,7 @@ class TransactionAssistance(DataSourceTrackedModel):
     uri = models.TextField(blank=True, null=True)
     cfda_number = models.TextField(blank=True, null=True, verbose_name="CFDA Number")
     cfda_title = models.TextField(blank=True, null=True, verbose_name="CFDA Title")
-    cfda = models.ForeignKey(CFDAProgram, models.DO_NOTHING, null=True)
+    cfda = models.ForeignKey(CFDA, models.DO_NOTHING, related_name="related_assistance", null=True)
```

Comment:
So change it to something like "related_assistance_cfda"? We could, it might be overkill, because that reverse accessor is already going to be on the CFDA object already. (i.e. CFDA.objects.first().related_assistance)